src/tenderly/deploy-verify-zns.ts

```diff
@@ -0,0 +1,50 @@
+import { deployZNS } from "../../test/helpers";
+import * as hre from "hardhat";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+
+
+export const deployVerifyZNS = async ({
+  governor,
+  governorAddresses = [ governor.address ],
+  adminAddresses = [ governor.address ],
+} : {
+  governor : SignerWithAddress;
+  governorAddresses ?: Array<string>;
+  adminAddresses ?: Array<string>;
+}) => {
+  const zns = await deployZNS({
+    deployer: governor,
+    governorAddresses,
+    adminAddresses,
+  });
+
+  // TODO: find a way to get these names from the object
```

Comment:
For a piece of feedback you left in my PR I added the string "ZNSDomainToken" to the constants list for when we deploy a new DomainToken contract. The same could be done here for each contract

Reply:
this is kind of what I did here, only these are not constants. I don't like this approach though.
it is weird to me that I can't seem to find a contract name in the contract object. No idea why they didn't include it. or maybe I'm just blind...